src/views/simplified-account/settings/card-payments/apple-pay.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Card payments
+{% endblock %}
+
+{% block settingsContent %}
+  {% if isReadOnly %}
+    {% include "includes/settings-read-only.njk" %}
+  {% endif %}
```

Comment:
I don't think these are necessary as `isReadOnly` is never passed in, and the page will simply `404` for a non-admin user - similarly on the other settings pages

Reply:
Removed